content/ADR-188-favorite-places.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: adr
+adr: 188
+title: Favorite Places
+date: 2023-02-07
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - davidejensen
+---
+
+## Abstract
+
+This document describes an approach on how to integrate the new Favorite Places functionality in Decentraland.
+
+## Glossary
+
+- **Place** A specific parcel or set of parcels identified by a [place UUID](/adr/ADR-186)
+- **Places** The structure of database and endpoints used in the places website
+
+## Problem
+
+To enhance the discoverability of the different Decentraland places, there is the need of adding a favorite mechanism. This will allow user to store and retrieve easily a set of their best loved places.
+To integrate this changes at multiple levels are required, from the visual representation in the explorer, to the backend structure that will allow the saving and retrieving of a user favorites.
+
+Following a list of the expected functionalities:
+
+- Users can add a place to their favorite from the explore menu
+- Users can navigate to a new tab listing all the places they added to favorites
+In a future iteration it will be possible to favorite places from the minimap HUD and also from the map parcel info pop-up, currently not possible due to technical limitations of the renderer map handling and Places API restriction to curated places.
+More in the limitations section.
+
+## Solution
+
+### Data storing
+
+After an initial discussion about whether to store favorites in the user profile or in Places, we are more incline to store everything in Places, more informations about the decision journey at the end.
+
+### Favorite retrieval
+
+The Places API will provide the required informations through:
+
+- An endpoint to obtain the (paginated if we decide for a big maximum amount) list of favorites.
```

Comment:
Pagination should be designed by default, we can't defer this to the future because it _will_ be needed and adding that complexity later on would be highly complex